app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cp, file, new_file = command.split()
```

Comment:
Before you must validate that command.split() have 3 elements
file, new_file bad variable name file != path

Reply:
Yes) i fixed it.